obstacles:

  center_x: [  9, -16,  -5, -15, -10, -11,  -3,    4,    9, -1.8, -17,   5.5,   17,  21, 13.5, 16.5, -22,   1, -10,  16,  2, -16, -13,   9,  -8,  4.5, 11, 15, -0.5, -21, 13.5,  4,  -5,   0, -13, -7, -13,  10,  -5,  -6]
  center_y: [ 11, 5.5,  12,  13,   7,  -1,   5,  -10, -3.5, -3.8,  -8,     4,   -3,   6,  -11,    9,  -3,   4,  -9,   4,  8,  -1,   9, -11,   3, -0.5,  2, -7,   -9,   3, -1.5, -5,   0, 0.5, 2.5, -4,  -5, 6.5,  -8, 8.5]
  radius:   [2.5, 2.2, 1.2,   1, 1.5, 1.5,   2,  1.5,  2.5,  2.2,   2,     2,  1.5,   2,  1.2,    2,   2, 1.5, 1.8, 2.5,  2,   2, 1.4, 2.5, 1.7,  1.6,  2,  2,  1.6, 1.7,  1.6,  2, 1.6, 1.5, 1.4,  2, 1.5, 1.5, 1.6, 1.8]

  obstacle_threshold: 0.3
  narrow_distance: 0.2
